src/main/java/thefarm/orchards/apple/model/Apple.java

```diff
@@ -0,0 +1,88 @@
+package thefarm.orchards.apple.model;
+
+public class Apple {
+    private String stem;
+    private String blossom;
+    private String peeling;
+    private String juice;
```

Comment:
Similar questions here, what would this be used to determine or do? Is it describing if the apple has juice in it? Has been juiced? Has potential to be juiced?

Reply:
It would be the potential to be messy, thirst quenching, and juiced

Reply:
In a production environment, a property which is intrinsic to the object but transient to the use of the project in any aspect ("messy" for example) is negligible.

Thirst quenching will be left to the marketing section of the app lol.

Juiced is of some potential. Describe the process of juicing an apple? My guess is that you'll want to add a property for something like ``consumed`` or ``viable`` or similar.

A process such as juicing, or eating or crushing, etc, would leave an apple consumed or unviable, etc, which means it may not be a property to use at this level.